### PR TITLE
Add Windows support

### PR DESCRIPTION
**Pull request by Mitchell-Riley**

### CODE REVIEW

File: gotools_settings.py [hunk 1]
```diff
@@ -77,8 +77,13 @@ def load_goenv():
 
   # Find the go binary on PATH, and abort initialization if it can't be found.
   gobinary = None
-  for segment in ospath.split(':'):
```
Comment (ironcladlou) [inline gotools_settings.py]: I should have used https://docs.python.org/3.0/library/os.html#os.pathsep, which would avoid the platform specific code you introduced for this.

**Comment by ironcladlou** [comment]
Thanks for the PR. What do you think about consolidating the platform executable extension suffixing into the ToolRunner class? It may not be the best abstraction but I'd rather have all the logic consolidated.

Unfortunately I won't be testing GoTools on Windows, so any improvements for that platform are going to have to come from contributors.

**Comment by Mitchell-Riley** [comment]
Sorry about that, I'm not much of a Python guy.

I can squash these commits if you prefer.

**Comment by ironcladlou** [comment]
>Sorry about that, I'm not much of a Python guy.

That makes two of us. :grin: 

>I can squash these commits if you prefer.

Yes, please squash them. I'll regression test on linux/osx a little later and then can merge this in. Thanks again!

**Comment by ironcladlou** [comment]
I added this to the 0.1.5 release. Thanks again for the contribution.

**Pull request merged**